### ISSUE TITLE
Library page scrollTo and a11y updates for interactive elements

### ISSUE THREAD

**Issue by amygroshek** [Hyperobjekt/untd-site#22]
- [ ] Resolve a11y warnings for the anchors that trigger library topic switching (one possibility is that these could be buttons, instead, to get around the issue).
```
/Users/lougro/repos/untd-site/src/pages/research-library.js
  148:19  warning  The href attribute is required for an anchor to be keyboard accessible. Provide a valid, navigable address as the href value. If you cannot provide an href, but still need the element to
resemble a link, use a button and change it with appropriate styles. Learn more: https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/anchor-is-valid.md  jsx-a11y/anchor-is-valid
  148:19  warning  Visible, non-interactive elements with click handlers must have at least one keyboard listener

jsx-a11y/click-events-have-key-events
  148:19  warning  Static HTML elements with event handlers require a role

jsx-a11y/no-static-element-interactions
```
- [ ]  Implement smooth scrolling to and display of selected topic when page is loaded with hash that corresponds to the topic. So that, for example, when `/research-library/#quality-education` is loaded, the page scrolls to the research library content section and that topic is displayed.
